models/db.py

```diff
@@ -1,9 +1,36 @@
-from gluon.tools import Auth
 db = DAL('sqlite://storage.sqlite',pool_size=1,check_reserved=['all'])
+from gluon.tools import Auth, Service, PluginManager
+
 auth = Auth(db)
 
+## This allows me to still be able to use the web2py auth module while still customising properly the fields
+db.define_table('auth_user',
+                Field('first_name', length=128, default=''),
+                Field('last_name', length=128, default=''),
+                Field('voters_registration_number',),
+                Field('email', length=128, default='', unique=True),
+                Field('password', 'password', length=512,readable=False, label='Password'),
+                Field('registration_key', length=512, writable=False, readable=False, default=''),
+                Field('reset_password_key', length=512, writable=False, readable=False, default=''),
+                Field('registration_id', length=512, writable=False, readable=False, default=''))
+
+#validators
+custom_auth_table = db['auth_user'] # get the custom_auth_table
```

Comment:
Please try check PEP8 "Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space." https://www.python.org/dev/peps/pep-0008/#inline-comments